parachains/runtimes/assets/statemine/src/weights/xcm/mod.rs

```diff
@@ -142,7 +142,13 @@ impl<Call> XcmWeightInfo<Call> for StatemineXcmWeight<Call> {
 		_dest: &MultiLocation,
 		_xcm: &Xcm<()>,
 	) -> XCMWeight {
-		assets.weigh_multi_assets(XcmFungibleWeight::<Runtime>::initiate_teleport())
+		// Hardcoded till the XCM pallet is fixed
+		let hardcoded_weight = Weight::from_ref_time(200_000_000 as u64).ref_time();
+		let weight = assets.weigh_multi_assets(XcmFungibleWeight::<Runtime>::initiate_teleport());
+		if (weight > hardcoded_weight) {
```

Comment:
@ruseinov 
could be `min` function used here?

Reply:
@bkontur This is std and it's not re-exported by sp_std. So preferably not

Reply:
@joepetrowski @KiChjang what do you say? I haven't seen min/max used anywhere or being re-exported.

Reply:
@ruseinov 
I see several usages of `sp_std::cmp::min`, but I dont have up-to-date code base now

Reply:
I don't have a strong opinion on it

Reply:
if we can use min/max - let's use it. I thought there was a reason not to.

Reply:
@bkontur fixed. I was having the same dilemma, but since I could not find many usages I assumed it might have something to do with `no_std` environments.